apps/docusaurus/docs/integration/wallet-adapter-for-wallets.md

```diff
@@ -41,3 +41,112 @@ If your wallet provides functionality that is not included, you should open a pu
 ### Add your name to the wallets list
 
 Once the package is published, create a pull request against the [aptos-wallet-adapter](https://github.com/aptos-labs/aptos-wallet-adapter) package and add your wallet name to the [supported wallet list](https://github.com/aptos-labs/aptos-wallet-adapter#supported-wallet-packages) on the README file as a URL to your NPM package.
+
+## AIP-62 Wallet Standard
+
+The AIP-62 Wallet Standard is a chain-agnostic set of interfaces and conventions that aim to improve how applications interact with injected wallets. Read more about it [here](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-62.md).
+
+### Why integrating with the AIP-62 wallet standard?
+
+:::note
+In the near future and as wallets onboard to the new standard, the wallet adapter will deprecate the legacy standard and keep only the AIP-62 wallet standard support.
+:::
+
+The AIP-62 wallet standard eliminates the current issues wallets have:
+
+- Relying solely on a dapp detecting process logic can create a race condition risk in the case the dapp loads before a wallet and the dapp is not aware of the new wallets
+- The legacy standard is deeply integrated within the Aptos wallet adapter, and any change can cause breaking changes for dApps and wallets, creating endless maintenance work by requiring a dApp or wallet to implement these changes.
+- The legacy standard supports only the legacy TS SDK input, types, and logic. That means that it doesn't enjoy the features and enhancements of the new TS SDK. In addition, the legacy TS SDK does not receive any more support or new features.
+
+In addition, a wallet can benefit from onboarding with the AIP-62 wallet standard by:
+
+- Wallet owns and controls its own interface and can easily update and provide new features without introducing any breaking change to the dapp
+- The wallet integration code lives in the wallet codebase and does not require a wallet to create and maintain another wallet package
+- Wallets dont need to rely on dapps to install and maintain its wallet package
+- The AIP-62 wallet standard provides better validation and error handling support, and uses the new TS SDK which is more reliable, fast and actively maintained and developed with new features (the legacy sdk, i.e aptos, is no longer actively maintained and developed).
```

Comment:
Since this is repeated from another section, maybe we can link instead of having two separate instances of the same info?

Reply:
I think it benefits have them in both places, so when we redirect dapps or wallets to each related page both can have it

Reply:
Ok sounds good. Just make sure they've incorporated the nits made to the other section